### ISSUE TITLE
MD5 sums mismatch

### ISSUE THREAD

**Issue by aleksey-orekhov** [ContinuumIO/anaconda-issues#733]
Many 32 bit windows packages have MD5 sums mismatch error when doing conda install:

logilab-common-1.0.2-py27_0
jupyter_client-4.11.1-py27_0
jupyter_core-4.0.6-py27_0
logilab-common-1.0.2-py27_0
sas7bdat-2.0.6-py27_0
cherrypy-3.8.0-py27_0
nbformat-4.0.1-py27_0
wheel-0.26.0-py27_1
pyflakes=1.0.0-py27_0
cython=0.23.4-py27_0

**Comment by aleksey-orekhov** [comment]
Error similar to #704

**Comment by ilanschnell** [comment]
Thanks for reporting this problem.  We would need more information in order to help.  What is the output from `conda info`?  Thanks.

**Comment by meereeum** [comment]
Same error message, different OS (linux-64) with conda v4.0.3

```Error: MD5 sums mismatch for download: https://repo.continuum.io/pkgs/free/linux-64/libdynd-0.7.2-0.tar.bz2 (d41d8cd98f00b204e9800998ecf8427e != e9269a41a8e4a30f5f133f885d45f131)```

**Comment by ilanschnell** [comment]
The hashsum `d41d8cd98f00b204e9800998ecf8427e` is the one for an empty file.  Conda 4.0.4 (which was only online for about an hour) has a bug which displayed this MD5 hashsum error for every file downloaded.  I verified that `e9269a41a8e4a30f5f133f885d45f131` is indeed the MD5 of `linux-64/libdynd-0.7.2-0.tar.bz2`.

Does `conda udpate conda` work?

**Comment by meereeum** [comment]
I'm on 4.0.3 (not 4.0.4) - same error with ```conda update conda```

```
The following packages will be UPDATED:

    conda: 4.0.3-py27_0 --> 4.0.5-py27_0
```
ultimately followed by
```
Error: MD5 sums mismatch for download: https://repo.continuum.io/pkgs/free/linux-64/conda-4.0.5-py27_0.tar.bz2 (d41d8cd98f00b204e9800998ecf8427e != fe7877ccfa3045c621700e6879c67462)
```

**Comment by kalefranz** [comment]
If you are running conda `4.0.3`, you unfortunately need to back yourself out of that version.  The easiest solution is to run

    conda list --revisions

Find which revision number upgraded you to conda `4.0.3`, and roll back to one revision before that...

    conda install --revision NUMBER

See [this](https://github.com/conda/conda/issues/2241#issuecomment-195430033) issue for more details.

You can also download the latest miniconda from [here](http://repo.continuum.io/miniconda/) and install over the top of your current conda install.  For example

    wget http://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
    bash Miniconda3-latest-MacOSX-x86_64.sh -bfp CURRENT_INSTALL_LOCATION

**Comment by meereeum** [comment]
Thanks @kalefranz , fixed by rolling back, updating to ```4.0.5```, and re-adding the other packages that I had installed since previous upgrade.
